src/utilities/calculateYearlyEarnings.ts

```diff
@@ -12,47 +12,48 @@ import BigNumber from 'bignumber.js';
 export const calculateYearlyEarningsCents = ({
   asset,
   isXvsEnabled,
-  borrowBalanceCents,
+  borrowBalanceCents = new BigNumber(0),
   supplyBalanceCents = new BigNumber(0),
   yearlyEarningsCents = new BigNumber(0),
 }: {
   asset: Asset;
-  borrowBalanceCents: BigNumber;
   isXvsEnabled: boolean;
+  borrowBalanceCents?: BigNumber;
   supplyBalanceCents?: BigNumber;
   yearlyEarningsCents?: BigNumber;
```

Comment:
I think it's confusing to pass these parameters given they aren't used in the calculation of the yearly interests.

Looking at this function, I would expect to pass `asset` only and be returned the yearly earnings only. Looking at the codebase, we don't use the extra parameters returned anyway (`supplyBalanceCents` and `borrowBalanceCents`) outside of the `calculateYearlyEarningsForAssets` function, which also passes and returns them but doesn't actually do anything with them either (apart from compounding them to return the total sums). I'd get rid of these completely and simplify this function so it only returns the yearly earnings generated by the asset passed through params.

Reply:
We do use those values in the `MyAccount` component. `borrowBalanceCents` should always equal `userBorrowBalance` so that is redundant. `supplyBalanceCents` is used in calculating the `netApy`

Reply:
Ok I see. then Perhaps `calculateYearlyEarningsForAssets` should be named to something else, because effectively it's not just calculating the yearly earnings but also returns more data.
I think we have to be very careful with these functions so their logic and names are as clear as water.

Reply:
Ok, yes I agree that a clear name is important. I'll update to `calculateYearlyEarningsAndSupplyBalanceForAssets` and we can look at a better solution for calculation Supply Balance in the future